botocore/handlers.py

```diff
@@ -596,6 +596,38 @@ def parse_get_bucket_location(parsed, http_response, **kwargs):
     parsed['LocationConstraint'] = region
 
 
+def document_s3_expires_shape(section, event_name, **kwargs):
```

Comment:
Can you include an example for what each of these gets rendered as?

Reply:
For the 'response-example' section it would look like this:
<img width="777" alt="syntax" src="https://github.com/user-attachments/assets/f2e43973-9987-400e-9e1b-127929dbce45">
For the 'response-params' section it would look like this:
<img width="626" alt="params" src="https://github.com/user-attachments/assets/dbeaf4ed-2b42-4b99-9d11-86fa5c668053">